types/aws-lambda/trigger/sqs.d.ts

```diff
@@ -44,3 +44,12 @@ export interface SQSMessageAttribute {
 export interface SQSMessageAttributes {
     [name: string]: SQSMessageAttribute;
 }
+
+// https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#services-sqs-batchfailurereporting
+export interface SQSEventResponse {
```

Comment:
This is a response dedicated for batches. so I would suggest to make this clear in the naming. `SQSBatchResponse` is used by the aws-sdk.
```suggestion
export interface SQSBatchResponse {
```